include/xtensor/xchunked_view.hpp

```diff
@@ -19,6 +19,21 @@
 namespace xt
 {
 
+    // SFINAE test if chunked
+    template <typename T>
+    class has_chunks
+    {
+    private:
+        typedef char YesType[1];
+        typedef char NoType[2];
+
+        template <typename C> static YesType& test(decltype(&C::chunk_shape));
+        template <typename C> static NoType& test(...);
+
+    public:
+        enum { value = sizeof(test<T>(0)) == sizeof(YesType) };
+    };
```

Comment:
This is redundant with the `is_chunked` function defined [here](https://github.com/xtensor-stack/xtensor/blob/5b7446890977ae287c6f8f7d3ae6203d19bca4f0/include/xtensor/xchunked_array.hpp#L288).

Maybe you need to add an overload with no arguments to be able to invoke it in `std::enable_if`